docs/pages/installation.mdx

```diff
@@ -160,9 +161,10 @@ repositories.
    has the same major version as the service that conducts automatic updates:
    
    ```code
-   $ export TELEPORT_PKG="teleport-ent teleport-ent-updater"
    $ export TELEPORT_DOMAIN=<Var name="example.teleport.com" />
-   $ export TELEPORT_VERSION="$(curl https://$TELEPORT_DOMAIN/v1/webapi/automaticupgrades/channel/default/version | sed 's/v//')
+   $ export TELEPORT_VERSION="$(curl https://$TELEPORT_DOMAIN/v1/webapi/automaticupgrades/channel/stable/cloud/version | sed 's/v//')"
+   $ export TELEPORT_PKG="teleport-ent-${TELEPORT_VERSION?} teleport-ent-updater"
```

Comment:
Ah, ran into an error here. When installing via apt, the command should be
`apt-get install teleport-ent=X.Y.Z` instead of `apt-get install teleport-ent-X.Y.Z`